src/wp-admin/includes/schema.php

```diff
@@ -713,6 +713,12 @@ function populate_options( array $options = array() ) {
  * @since 2.0.0
  */
 function populate_roles() {
+	global $wp_roles;
```

Comment:
```suggestion
 $wp_roles = wp_roles();
```
This is better to do this? To ensure that global is set?

Reply:
Thanks for the suggestion @spacedmonkey! 
Using just `$wp_roles = wp_roles();` is indeed cleaner and ensures the roles object is always properly initialized. 
I have updated the PR accordingly

Reply:
I tested the patch again, and we’re getting the same results.